src/notaryUtils.js

```diff
@@ -7,16 +7,24 @@ const getTransferState = require('./transferUtils').getTransferState
 const Payments = require('./payments')
 
 /**
+ * @returns {String} New case ID
+ */
+function createCaseID () {
+  return uuid()
+}
+
+/**
  * @param {Object} params
  * @param {URI} params.notary
  * @param {Condition} params.receiptCondition
  * @param {[Payment]} params.payments
  * @param {String} params.expiresAt
+ * @param {String} params.caseID
  * @returns {Promise<URI>} Case ID
  */
 function setupCase (params) {
   return co(function * () {
-    const caseID = params.notary + '/cases/' + uuid()
+    const caseID = params.notary + '/cases/' + (params.caseID || uuid())
```

Comment:
We should add validation to the passed in caseID here, because the database in the notary is currently only setup
to handle 40 characters.

https://github.com/interledger/five-bells-notary/blob/176b9c7eb744eb857bec16bb6b412c1c2ce62628/migrations/201601181500_create_cases.js#L17